support-frontend/assets/components/subscriptionCheckouts/address/deliveryCheck.js

```diff
@@ -0,0 +1,48 @@
+// @flow
```

Comment:
Regarding file location: do we have a convention of this kind of file being in a `helpers` directory or similar?

Reply:
helpers is one of my words to be avoided at all costs 😆 , this is because it includes more stuff than it excludes i.e. it's generic.  We should really group things according to cohesive functionality where possible. It does make organising things take significantly more thought, but it's a better time to do that thought than when trying to refactor later.